test/cmocka/include/power_tables.h

```diff
@@ -0,0 +1,155 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ */
+
+/* Include Files */
+#include <string.h>
+
+/* Reference table generated by Matlab power() */
+static const double power_table[64][6] = {
```

Comment:
@singalsu I think we want to make sure all our const coeff data or maths data is always aligned on cache line size for HiFi3 etc. I know this is optimal for x86 SIMD, and I dont think we are doing this today.

Reply:
I wonder how to achieve it. Is there some pragma available? It's nearly impossible to do and maintain manually without creating holes.

Reply:
Do I need to do anything here ?

Reply:
@ShriramShastry nothing needed atm. @singalsu its worth checking for each HiFi SIMD version what the needed alignment is. i.e. its usually data frame size (size of all data elems used in single instruction).